### DISCUSSION TITLE
Hi, my render takes a lot of time in a cloud function

### DISCUSSION THREAD

**Discussion by garciapd** [frinyvonnick/node-html-to-image#167] [General]
Hi, i'm using Google cloud functions and generating the images takes really long.

Any hint or config to reduce the latency?

Thanks

**Comment by frinyvonnick** [comment]
Hello @garciapd 👋 

node-html-to-image use the default puppeteer. Here is a [message](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteer-vs-puppeteer-core) in which I talk about puppeteer-core it might help you :)
